README.md

```diff
@@ -57,7 +57,7 @@ Quick Start
 -----------
 
 ```bash
-pip3 install green    # To upgrade: "pip3 install --upgrade green"
+pip3 install green    # To upgrade: "pip3 install --upgrade green"; To include pyproject.toml support in python <= 3.11 "pip3 install green[toml]"
```

Comment:
This makes for a very long comment. I would probably split the whole thing. Note that 3.11 does include toml support. Note that some shells such as zsh will require the brackets to be quoted.

```suggestion
pip3 install green
# To upgrade:
pip3 install --upgrade green
# To add pyproject.toml support in python < 3.11:
pip3 install 'green[toml]'
```

Quotes for zsh because:
```
> echo green[toml] 
zsh: no matches found: green[toml]
> echo 'green[toml]'
green[toml]
```
With bash it does work:
```
bash-3.2$ echo green[toml]
green[toml]
```